src/lib/digest_service/digest_data_aggregator.py

```diff
@@ -94,14 +94,20 @@ def _get_error_comments(
             )
 
             # construct error comment based on the job run URL and error message
+            error_message = resource_run.get("error_message", "Unknown errors")
             if job_run_url:
-                error_comment = f" - <a href={job_run_url}>ERROR: {resource_run['error_message'][:50]}</a>"
-            elif resource_run["error_message"]:
-                # if no job run, use the error message directly
-                error_comment = resource_run["error_message"]
+                # truncate error message if > 100 chars
+                truncated_message = (
+                    (error_message[:100] + "...")
```

Comment:
maybe, it make sense to define max error length text as a constant (if though we still can keep the constant in this module)

Reply:
yes, sure. also had this idea in mind